studio/schemas/documents/benefit.ts

```diff
@@ -1,27 +1,45 @@
-import { defineType } from 'sanity';
-import { richText, title } from '../fields/text';
+import { defineField, defineType } from "sanity";
+import { richText, title } from "../fields/text";
 
 export const benefitId = "benefit";
+export const benefitTypeId = "benefitType";
+
+const BENEFIT_TYPES = [
+  { title: "Gadgets", value: "gadgets" },
```

Comment:
Let's also add:
  { title: "None", value: "none" },
  
  So that we can allow users to have benefit sections without any visuals like graphs

Reply:
The user already has the option to leave the field blank, so that the document does not have the benefitType property. But an explicit option is probably a good idea :+1:

Reply:
Yea, I think that gives a better UX as it has an actual "none" value

Reply:
Is "None" the right word though? What about "Generic" or "Other"?

Reply:
Yes, I believe so. The user don't want the benefit to have a type so it chooses "None"